src/post/content.unit.test.ts

```
@@ -20,6 +21,82 @@ describe('ContentPreparer', () => {
             addMentions: false as const,
         };
 
+        describe.only('Removing gated content', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)